src/blog/_posts/2018-01-31-opera-51.md

```diff
@@ -0,0 +1,175 @@
+  for `<video>` and `<audio>` elements has been changed to `metadata` in order to reduce bandwidth and resource usage,
+  by only loading resource metadata and not the media resource itself. This aligns Opera with other browser
+  implementations.
+* A `"NotSupportedError"` `DOMException` is now thrown [when](https://googlechrome.github.io/samples/media/playback-rate-exception)
+  a media element&#8217;s [playbackRate](https://html.spec.whatwg.org/multipage/media.html#dom-media-playbackrate) is set to a value
+  that is [not supported](https://developers.google.com/web/updates/2017/12/chrome-63-64-media-updates#unsupported-playbackRate-raises-exception)
+  by Opera, like a negative value.
+* [`getUserMedia()`](https://w3c.github.io/mediacapture-main/#dom-mediadevices-getusermedia()) now returns a rejected
+  Promise with a `DOMException` or `OverconstrainedError` when there is an error.
+
+### Blink > Network
+
+* The [cache mode](https://fetch.spec.whatwg.org/#concept-request-cache-mode) of a `Request` can now be specified using
+  the [`cache`](https://fetch.spec.whatwg.org/#dom-request-cache) option.
+* The cache mode of a `Request` can now be accessed using
+  [`Request.prototype.cache`](https://developer.mozilla.org/en-US/docs/Web/API/Request/cache). Reload requests, the
+  `"reload"` value, are now also exposed through said property.
+
+### Blink > Permissions API
+
+* The [Permissions API](https://w3c.github.io/permissions/) can now query the status of the
+  [`camera`](https://w3c.github.io/permissions/#dom-permissionname-camera) and
+  [`microphone`](https://w3c.github.io/permissions/#dom-permissionname-camera) permissions.
+
+### Blink > Scroll
+
+* [`HTMLElement.prototype.focus`](https://html.spec.whatwg.org/multipage/interaction.html#dom-focus), accepts the
+  `preventScroll` option, to allow focusing an element without scrolling to it.
+
+### Blink > SVG
+
+* Support for the [`transform-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-box) CSS property has
+  been added. This allow developers better control over what percentages are resolved against in the
+  [`transform`](https://drafts.csswg.org/css-transforms/#transform-property) and
+  [`transform-origin`](https://drafts.csswg.org/css-transforms/#transform-property) properties on SVG elements.
+
+### Blink > WebAudio
+
+* [`AudioWorklet`](https://webaudio.github.io/web-audio-api/#AudioWorklet), an API that
+  [exposes](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/) low-level audio processing capability
+  to support custom AudioNodes, is now available behind the experimental flag.
+
+### Blink > WebRTC
+
+* The [`RTCPeerConnection` interface](https://w3c.github.io/webrtc-pc/#interface-definition) now supports `addTrack()`
+  for single stream use cases, as well as `removeTrack()`, `getSenders()` and `ontrack`. A minimal version of the
+  `RTCRtpSender` interface is now also supported.
+
+### Blink > WindowDialog
```

Comment:
I guess these are crbug components but they work as titles as well, except maybe this one.

Reply:
Renaming to "Window APIs"